### PR TITLE
Initial commit full calc app 

### PR DESCRIPTION
**Pull request by rashkov09**

### CODE REVIEW

File: app/src/test/java/com/tutorial/calculatorapp/model/Calculator.kt [hunk 1]
```diff
@@ -0,0 +1,98 @@
+package com.tutorial.calculatorapp.model
+
+import android.view.View
+import android.widget.Button
+import com.tutorial.calculatorapp.`interface`.ICalculator
+
+class Calculator : ICalculator {
```
Comment (Legementarion) [inline app/src/test/java/com/tutorial/calculatorapp/model/Calculator.kt]: So you have created it only for tests?  Could we reuse it on MainActivity?

Comment (Legementarion) [inline app/src/test/java/com/tutorial/calculatorapp/model/Calculator.kt]: So you have created it only for tests?  Could we reuse it on MainActivity?

File: app/src/test/java/com/tutorial/calculatorapp/model/Calculator.kt [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    private var lastNumeric: Boolean = false
+    private var dotInserted: Boolean = false
+    private var isNegative: Boolean = false
+    private var hasResult = false
+
+
+
+    override fun onEqual(input: String) : Double {
+        var result = 0.0
```
Comment (Legementarion) [inline app/src/test/java/com/tutorial/calculatorapp/model/Calculator.kt]: Lets move all values, numbers and strings to constants

File: app/src/test/java/com/tutorial/calculatorapp/model/Calculator.kt (hunk 1 above)
Comment (rashkov09) [reply app/src/test/java/com/tutorial/calculatorapp/model/Calculator.kt]: Yes, I removed the logic from MainActivity and used Calculator class.

File: app/src/test/java/com/tutorial/calculatorapp/model/Calculator.kt (hunk 2 above)
Comment (rashkov09) [reply app/src/test/java/com/tutorial/calculatorapp/model/Calculator.kt]: Moved them to constants

File: app/src/test/java/com/tutorial/calculatorapp/model/Calculator.kt (hunk 1 above)
Comment (rashkov09) [reply app/src/test/java/com/tutorial/calculatorapp/model/Calculator.kt]: Yes, I removed the logic from MainActivity and used Calculator class.

**Review by Legementarion — APPROVED** [review]
LGTM

**Pull request merged**